pyspcwebgw/__init__.py

```diff
@@ -99,23 +99,35 @@ async def _async_ws_handler(self, data):
         _LOGGER.debug("SIA code is %s for ID %s", sia_code, spc_id)
 
         if sia_code in Area.SUPPORTED_SIA_CODES:
-            entity = self._areas.get(spc_id, None)
+            entities = [self._areas.get(spc_id, None)]
             resource = 'area'
         elif sia_code in Zone.SUPPORTED_SIA_CODES:
-            entity = self._zones.get(spc_id, None)
+            entities = [self._zones.get(spc_id, None)]
             resource = 'zone'
+        elif sia_code in ('CL', 'OP'):
```

Comment:
Wouldn't it be nice to move CL and OP to a list like Area.SUPPORTED_SIA_CODES_ALL_AREAS, and use that here?
Because now we have Area-SIA codes that are supported, but not listed among the supported sia codes in the Area.py file.

Reply:
I think this comment is still valid

Reply:
...but maybe it is better to let it be like this for now, and make small cosmetic changes later.

Reply:
...but maybe it is better to let it be like this for now, and make small cosmetic changes later.

Reply:
Yes, please address this in a separate PR once the dust has settled.